apps/kudos-react-app/src/route-helpers/route-renderer.tsx

```diff
@@ -0,0 +1,64 @@
+import { KudosRoute } from "@kudos/kudos-types";
+import {
+  RouteObject,
+  RouterProvider,
+  createBrowserRouter,
+} from "react-router-dom";
+import Dashboard from "../routes/dashboard";
+import ErrorPage from "../routes/error";
+import Home from "../routes/home";
+import Login from "../routes/login";
+import Me from "../routes/me";
+import { ProtectedRoute } from "./protected-route";
+
+export const kudosRoutes: KudosRoute[] = [
+  {
+    path: "/",
+    element: <Home />,
+    name: "home",
+    errorElement: <ErrorPage />,
+    children: [
+      {
+        path: "/login",
+        name: "login",
+        element: <Login />,
+        errorElement: <ErrorPage />,
+      },
+      {
+        path: "/me",
+        name: "me",
+        element: (
+          <ProtectedRoute>
+            <Me />
+          </ProtectedRoute>
+        ),
+        errorElement: <ErrorPage />,
+      },
+      {
+        path: "/dashboard",
+        name: "dashboard",
+        element: (
+          <ProtectedRoute>
```

Comment:
This is how we would protect a route. Meaning it would redirect the user if they are not authenticated.